spyder/plugins/editor/widgets/tests/test_codeeditor.py

```diff
@@ -366,5 +366,144 @@ def test_brace_match(editorbot):
         assert editor.bracepos == expected
 
 
+def test_editor_backspace_char(editorbot):
+    """Regression test for issue spyder-ide/spyder#12663."""
+    qtbot, editor = editorbot
+    text = "0123456789\nabcdefghij\n9876543210\njihgfedcba\n"
+    editor.set_text(text)
+    expectedCol = 7
```

Comment:
Please rename this variable to `expected_column` because we only camel case variables to name classes.